doc/Caliptra.md

```diff
@@ -42,33 +44,32 @@ The Caliptra Workgroup acknowledges the following individuals for their contribu
 
 <div style="page-break-after: always"></div>
 
-**Revision Table**
+**Revisions**
 
 |**Date**|**Revision #**|**Author**|**Description**|
 | - | - | - | - |
-|February 2022|0.2|Prabhu Jayana (AMD)</p> Bryan Kelly (Microsoft)</p> Andrés Lagar-Cavilla (Google)</p> Jeff Andersen (Google)</p> | Initial proposal draft |
-|March 2022|0.4| Rob Strong (AMD)</p> Prabhu Jayana (AMD)</p>| Migrated to OCP template</p>Edits to clarify language use and added sections related LifeCycle support, fuse, crypto requirements, Kat support, etc.</p> |
-|April 12, 2022|0.5|Rob Strong (AMD)|Various edits and formatting modifications to get to v.5|
-|June 2022|0.51|Rob Strong (AMD)|Updated the FW Signing/Verification Algorithms section - added references to OCP Secure Boot specification.</p>Updated Physical Attack Countermeasures - updated the section to reference NiST paper ([[7](#ref-7)]) that discusses SCA as well as their countermeasures.|
-|June 2022|0.52|Nathan Nadarajah (AMD) Sudhir Mathane (AMD)|Added Threat Model section and initial content |
+|February 2022|0.2|Prabhu Jayana (AMD)</p> Bryan Kelly (Microsoft)</p> Andrés Lagar-Cavilla (Google)</p> Jeff Andersen (Google)</p> | Created initial proposal draft. |
```

Comment:
Are the intermediate revisions really needed? Not sure if anyone cares about the progression once 1.0 is published.

Reply:
+1, I think we could drop these. This table stopped being updated when we moved to GitHub.

Reply:
Out of scope for this commit. Please file an issue.

Reply:
Filed as #77.